packages/react-components/src/components/chart/baseChart.tsx

```diff
@@ -162,6 +163,18 @@ const BaseChart = ({ viewport, queries, size = { width: 500, height: 500 }, ...o
     ...chartEventsHandlers,
   };
 
+  const handleMouseDownCapture = (e: MouseEvent) => {
+    const target = e.target;
+
+    /* Condition to check localName of Canvas to stop onMouseDouwnCapture event 
+      propagation to fix widget dragging or moving issue while panning on chart */
+    if (target instanceof Element) {
```

Comment:
`target instanceof Element && target.localName === 'canvas'`

Reply:
Will this not stop all the mouse down capture on the chart section ?

Reply:
My bad, yes there is an issue with removing the trend cursor. onMouseDownCapture is stopping the event. I will look into it.

Reply:
Resolved the issue with onMouseDown event, and updated the recorded video.

Reply:
Do you mind testing all the features such as adding a TC, deleting, Zoom , panning ? just to be sure

Reply:
@tracy-french : the current solution still has issues, will work with @gmuralidharreddy and see if we can find a better solution

Reply:
Got it! Sounds good!